includes/class-fs-logger.php

```diff
@@ -125,6 +128,14 @@ function on() {
 			self::hook_footer();
 		}
 
+        /**
+         * Turn off instance
+         *
+         * @return void
+         */
+        public function off() {
```

Comment:
This method won't be needed anymore (please see my suggestion below). Also just a code convention thingy, we add a blank line after a function/method declaration.